### PR TITLE
Cometd test mock

### PR DESCRIPTION
**Pull request by brendan-hofmann**

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
DDF-294 CometD mock server and test fixes for search UI regression testing.

**Comment by mrmateo** [comment]
Hey Brendan, have you submitted an ICLA yet? http://www.codice.org/contributing

**Comment by brendan-hofmann** [comment]
Yes, Kit Plummer received it.

**Comment by mrmateo** [comment]
OK, thanks!

File: search-ui/standard/Gruntfile.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-sed');
     grunt.loadNpmTasks('grunt-express');
     grunt.loadNpmTasks('grunt-casperjs');
-    grunt.registerTask('test', ['express:test','casperjs']);
+    grunt.registerTask('test', ['express:test','casperjs','express-keepalive']);
```
Comment (stustison) [inline search-ui/standard/Gruntfile.js]: I'm not sure that we can actually use express-keepalive. I believe that will cause express to continue running indefinitely. We only want express to run until the casperjs tests have finished and then quit.

**Comment by brendan-hofmann** [comment]
Not sure why my machine is all twisted up, but uninstalling global grunt (even though they're the same version) got it working without express-keepalive. This should stop it from breaking the build.

**Pull request merged**